### PR TITLE
using Directory.Build.props to enforce consistency in package props

### PR DESCRIPTION
**Pull request by MikeBairdRocks**
Added `Directory.Buid.props` to enforce DRY principle and easier to maintain package properties throughout all projects.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/PiranhaCMS/piranha.core?pullRequest=1438) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/PiranhaCMS/piranha.core?pullRequest=1438) before we can accept your contribution.<br/><br/>:x: MikeBairdRocks [sign now](https://cla.dotnetfoundation.org/PiranhaCMS/piranha.core?pullRequest=1438)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/PiranhaCMS/piranha.core?pullRequest=1438) it.</sub>

**Comment by tidyui** [comment]
Quick question, I guess PackageId isn’t needed then if it’s the same as the project name. How about documentation file, isn’t that needed either?

**Comment by MikeBairdRocks** [comment]
Yes, I should of mentioned that I removed the duplication of [PackageId](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packageid) since it will use the AssemblyName.

For documentation, `GenerateDocumentationFile` is the equivalent to `DocumentationFile`. See the docs: [Document your code with XML comments](Document your code with XML comments). Also note that I made that option false in the test and examples projects to reduce the warnings in the build output.

File: Directory.Build.targets [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
   </PropertyGroup>
   <ItemGroup>
-    <EmbeddedFiles Include="$(GeneratedAssemblyInfoFile)"/>
+    <EmbeddedFiles Include="$(GeneratedAssemblyInfoFile)" />
+    <None Include="$(SolutionDir)piranha.png" Pack="true" Visible="false" PackagePath="" />
```
Comment (MikeBairdRocks) [inline Directory.Build.targets:7]: I moved the png to the sln level and this will include the png into every project.

**Pull request merged**